Makefile

```diff
@@ -23,21 +23,21 @@ down: ## turn this thing off
 	docker-compose down
 
 up: ## run this thing
-	docker-compose up -d
+	docker-compose up redis db seleniarm web -d
 
 up_milsch: ## run this thing
-	PARTNER=milsch docker-compose up -d
+	PARTNER=milsch docker-compose up redis db seleniarm web -d
 
 up_honors: ## run this thing
-	PARTNER=honors docker-compose up -d
+	PARTNER=honors docker-compose up redis db seleniarm web -d
 
 up_sset: ## run this thing
-	PARTNER=sset docker-compose up -d
+	PARTNER=sset docker-compose up redis db seleniarm web -d
```

Comment:
if we have folks on x64, we might want to make this configurable. this is probably fine for now though!